codebase2/codebase-sqlite/sql/schemas/v1.sql

```diff
@@ -0,0 +1,241 @@
+-- v1 codebase schema
```

Comment:
The same as the v2 schema, but with `1` as the schema version. I added this for historical record.